.travis.yml

```diff
@@ -1,14 +1,16 @@
 language: go
 
 go:
-  - 1.8
```

Comment:
this fails for some reason...

Reply:
likely related:
- https://github.com/alecthomas/gometalinter/issues/358
- https://github.com/golang/go/issues/21712

Reply:
Can you add a TODO and link to appropriate issues?

Reply:
@jlewi done adding notes